test/files/noformat2.jl

```diff
@@ -2050,6 +2050,11 @@ end
                b"""
         @test fmt("export a,b", 4, 1) == str
 
+        str = """
+        public a,
+               b"""
+        @test fmt("public a,b", 4, 1) == str
```

Comment:
I would add a testset to test/default_style.jl. The above one is good, I would just expand on it to also go with

```
@testset "public keyword" begin
    # test you wrote above

    str = """public a, b"""
    @test fmt("public a,b", 4, 100) == str
end

```

Reply:
after this one

https://github.com/domluna/JuliaFormatter.jl/blob/master/test/default_style.jl#L4744-L4757

Reply:
the file you edited doesn't actually have any tests in it run. It's a old version of test/default_style.jl that was copy/pasted to make sure if you add a no format tag at the top it wouldn't do anything to the file.

Reply:
eh whoops! will fix shortly :D

Reply:
OK hopefully fixed, thanks!